roles/senior_reliability_engineer.md

```diff
@@ -0,0 +1,155 @@
+
+Other notable things:
+
+- Every Friday afternoon is dedicated to learning new skills
+- Everyone is encouraged to write blog posts regularly
+- Our handbook is open sourced
+- We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
+- Retreats and trips every year
+
+## Salary
+
+This role has a salary of £60-75k depending on experience.
+
+
+## Expectations
+
+### To drive the reliability engineering agenda across Made Tech
+
+We expect the Senior Reliability Engineers to drive the reliability engineering agenda across the business. They should set the standard for what "good" looks like for applications from a maintainability, scalability, and security perspective, and ensure delivery teams understand are bought in to these standards.
+
+### To coach and nurture team members to improve their reliability engineering capability
+
+We expect our Senior Reliability Engineers to proactively provide thoughtful and meaningful feedback for their team. A Senior Reliability Engineer is expected to spend time helping team members to improve their skills. A Senior Reliability Engineer is expected to identify and nurture candidates for Senior Reliability Engineer positions.
+
+## Reliability Engineer Expectations
+
+### To be ambassadors of reliability engineering culture
+
+We expect our Reliability Engineers to contribute to our reliability engineering culture, standards and thought leadership. We expect Reliability Engineers to champion existing standards by working closely with delivery teams to coach and mentor them. We also expect all Reliability Engineers to be involved in discussions around improving our standards.
+
+### To proactively identify technical improvements and recommend them to delivery teams
+
+We expect our Reliability Engineers to be able to spot areas for improvement in codebases such as test suites, dependency upgrades, performance issues detected by monitoring. We also expect them to be able to highlight these issues in a jargon free way and be able to convey the relative urgency of the issues ultimately leading to codebases being improved.
+
+### To autonomously deliver working software
+
+We expect our Reliability Engineers to be able to work on entire features, from conception through to production deployment, without guidance and direction from others. This doesn't mean they have to be delivered without pairing, or asking advice from the wider team. What it does mean is the primary force for ensuring the feature is built and deployed in a timely fashion is the responsiblity of the Reliability Engineer working on the problem.
```

Comment:
Is feature the right language for Reliability Engineers? In my head RE shouldn't be about delivering features